docs/SOPs/1-updating_fedefl.md

```diff
@@ -0,0 +1,17 @@
+# Updating the Federal Elementary Flow List
+
+## Purpose
+
+This SOP documents procedures for making changes to the Federal Elementary Flow List (FEDEFL) data objects on the Federal LCA Commons (FLCAC).
+
+## Procedure
+
+1. Any changes to the FEDEFL must align with a fedelemflowlist package release and update to the list version.
```

Comment:
For someone not intimately familiar with the updating process, a little more context on this sentence would be helpful.